coldfront/templates/email/project_join_request_approved.txt

```diff
@@ -1,31 +1,56 @@
-Dear {{ user.first_name }} {{ user.last_name }},
-
+Dear {{ user.first_name }} {{ user.last_name }},                                                                                                                                                                    
```

Comment:
Can you remove this trailing whitespace? There are also some single trailing spaces at various points below.